packages/discovery-react-components/src/utils/document/__tests__/transformEnrichment.spec.tsx

```diff
@@ -0,0 +1,82 @@
+import transformEnrichment from '../transformEnrichment';
+import isEqual from 'lodash/isEqual';
+import contractData from 'components/CIDocument/components/CIDocument/__fixtures__/contract.json';
+import invoiceData from 'components/CIDocument/components/CIDocument/__fixtures__/invoice-index_op.json';
+import purchaseOrderData from 'components/CIDocument/components/CIDocument/__fixtures__/po-index_op.json';
+import { EnrichedHtml } from 'components/CIDocument/types';
+
+describe('transformEnrichment', () => {
+  let transformedDoc: EnrichedHtml[];
+  const enriched_html = contractData.enriched_html;
+
+  beforeAll(() => {
+    // trasnform doc for use in tests
+    transformedDoc = transformEnrichment(enriched_html);
+  });
+
+  it('takes an array, and returns an array', () => {
+    expect(Array.isArray(enriched_html)).toBeTruthy();
```

Comment:
hopefully we can do away with checks like these once we are able to type check our JSON blobs. I believe the work being done [here](https://github.ibm.com/Watson-Discovery/disco-widgets/issues/468) would also relate to our JSON blobs used in unit testing

Reply:
This issue was about the returning type, so I have added this test explicitly. I think it is good to have this extra layer.

Reply:
Actually, this does bring up a good point: the return type was already an array (`EnrichedHtml[]`), yet we were returning an object. But TS didn't call it out.